charts/argo-cd/templates/NOTES.txt

```diff
@@ -9,7 +9,8 @@ In order to access the server UI you have the following options:
       - Add the `--insecure` flag to `server.extraArgs` in the values file and terminate SSL at your ingress: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/ingress.md#option-2-multiple-ingress-objects-and-hosts
 
 
-After reaching the UI the first time you can login with username: admin and the password will be the
-name of the server pod. You can get the pod name by running:
+After reaching the UI the first time you can login with username: admin and the random password generated during the installation. You can find the password by running
```

Comment:
```suggestion
After reaching the UI the first time you can login with username: admin and the random password generated during the installation. You can find the password by running:
```

Reply:
Grml, we now need to reword the commit message :-( The DCO bot complains about missing `Signed-off-by: Vale <valerauko@gmail.com>`. Would you mind to reword this?

Reply:
I fixed it myself :-)